src/app/pages/users/users-create-page/users-create-page.component.ts

```diff
@@ -124,18 +127,18 @@ export class UsersCreatePageComponent implements OnInit {
     );
   }
 
+  get baseOrganizationNameIndentation () {
```

Comment:
You have same function in file `src/app/pages/organizations/organizations-create-page/organizations-create-page.component.ts`, so put it in common file (perhaps item.ts)

Reply:
I can't put it in `item.ts` because I need the full context (i.e. the root organization) which is hold by the component. But I agree this solution is not satisfying. This could be some sort of mixin or utility function, but I thought it was too early to extract this (i.e. a simple method only used twice). I can reconsider my position though.

Reply:
ok, can be update later with many items types ^_^